core/src/main/res/values/strings.xml

```diff
@@ -489,6 +489,7 @@
     <string name="pref_smart_mark_as_played_disabled">Disabled</string>
     <string name="pref_image_cache_size_title">Image Cache Size</string>
     <string name="pref_image_cache_size_sum">Size of the disk cache for images.</string>
+    <string name="documentation_support">Documentation & Support</string>
```

Comment:
Maybe I need to escape this character with `\` or use `&amp;` instead?

Reply:
I think it should be `&amp;`